store/tikv/snapshot.go

```diff
@@ -114,6 +117,9 @@ func (s *tikvSnapshot) batchGetKeysByRegions(bo *Backoffer, keys [][]byte, colle
 }
 
 func (s *tikvSnapshot) batchGetSingleRegion(bo *Backoffer, batch batchKeys, collectF func(k, v []byte)) error {
+	sender := NewRegionRequestSender(s.store.regionCache, s.store.client)
+	sender.SetIsolationLevel(s.isolationLevel)
```

Comment:
I prefer to put `isolationLevel ` in `NewRegionRequestSender`, so we can easily check whether we have missed a `sender.SetIsolationLevel(s.isolationLevel)`?